cpu/cortexm_common/ldscripts/cortexm_base.ld

```diff
@@ -236,4 +236,8 @@ SECTIONS
     .end_fw (NOLOAD) : ALIGN(4) {
```

Comment:
In particular (from looking at occurrences of `end_fw`), this puts it in conflict with `flashpage_first_free` -- as `flashpage_first_free` looks at `end_fw`, pages used in `flash_writable` would also show up in `flashpage_first_free`. I see this as a successor to `flashpage_first_free` anyway, but they should still not conflict. If `end_fw` is not used anywhere else, it might suffice to swap them around so that flash_writable is before end_fw.

(My impression from `git praise` is that `end_fw` is exclusively used for `flashpage_first_free` -- if not, additional questions would be raised as to whether statically initialized content does actually get flashed.)

Reply:
Yes I also see this approach as a successor for the `flashpage_*_free` mechanism. I will deprecate these functions if this approach work fine. Therefore swapping the symbols around seems reasonable to me